contracts/access/manager/AccessManager.sol

```diff
@@ -52,7 +52,8 @@ contract AccessManager is Context, Multicall, IAccessManager {
     using Time for *;
 
     struct AccessMode {
```

Comment:
Should we rename that structure ?

Reply:
Yes! This name was left over from when it was the AccessMode enum :sweat_smile:

Reply:
`ContractConfig` or something?

Reply:
I did that, but I'd like to consider replacing "Contract" with "Target".

- `setContractFunctionGroup` → `setTargetFunctionGroup`
- `isContractClosed` → `isTargetClosed`
...

WDYT?

Reply:
Definitely, I like that.